### PR TITLE
Create 'Add an agent' card

### PR DESCRIPTION
**Pull request by eddie-m-m**
- Create 'Add an agent' card
- Create 'Add an agent' button in card

### CODE REVIEW

File: src/pages/Search.tsx [hunk 1]
```diff
@@ -44,6 +44,17 @@ export default function Search() {
       <div>
         {data &&
           data.map((r) => <SearchResult result={r as Agent} key={r.id} />)}
+        <Card body border="light" className="shadow text-center">
```
Comment (Timmehs) [inline src/pages/Search.tsx:47]: To match the flat style in the mobile app (note removing the `body` prop:
```suggestion
        <Card border="0" className="text-center">
```

File: src/pages/Search.tsx [hunk 2]
```diff
@@ -44,6 +44,17 @@ export default function Search() {
...
+          <h3 className="m-3 pt-4">Can't find what you're looking for?</h3>
+          <Button
+            aria-label="Add an agent"
+            size="lg"
+            variant="primary"
+            className="m-4 w-75 p-3 text-light"
+          >
+            Add an agent
+          </Button>
```
Comment (Timmehs) [inline src/pages/Search.tsx:56]: Unlike the other search results cards, we'll want to specify the body here so we can move that custom size-4 padding to the parent component here (loose guideline is padding on containers, margin on children).

I'm also removing the margin and padding from the button and changing the text to white
```suggestion
          <Card.Body className="p-4">
            <h3 className="mb-4">Can't find what you're looking for?</h3>
            <Button
              aria-label="Add an agent"
              variant="primary"
              size="lg"
              className="w-75 text-light"
            >
              Add an agent
            </Button>
          </Card.Body>
```

**Review by Timmehs — CHANGES_REQUESTED** [review]
Great work on this! The card was looking a little chunky to me compared to the other search result cards, so I had a play with some of the styles, let me know if the suggested changes don't make sense or if I can clarify the thinking at all.

Here's the before/after:
## Before
image

## After

image

File: src/pages/Search.tsx (hunk 1 above)
Comment (eddie-m-m) [reply src/pages/Search.tsx:47]: This style makes so much more sense to me. It smoothly meshes with the agent cards. And dropping the shadow adds enough distinction to differentiate it subtlety from the list of agent cards.

File: src/pages/Search.tsx (hunk 2 above)
Comment (eddie-m-m) [reply src/pages/Search.tsx:56]: Sounds good.

**Review by Timmehs — APPROVED** [review]

**Pull request merged**